devel/ansible/roles/bodhi/tasks/rabbitmq.yml

```diff
@@ -6,6 +6,12 @@
   with_items:
       - rabbitmq-server
 
+- name: Start rabbitmq
+  service:
+    name: rabbitmq-server
+    state: started
+    enabled: yes
```

Comment:
This should probably be moved below the copy of the `override.conf` file below, because systemd will make use of it.

Reply:
While trying to upgrade Vagrant to F31 I discovered that trying to start the server after the limits override and the plugin activation makes it crash. I had to firstly start the server and then do the customization.
I did not try if that happens also in F30: I could check, but both the limits override and the plugin activation triggers a service restart with a `systemd daemon-reload`.

For the same reason I changed the user creation step: with the `permissions` command the step failed with a strange `unknown host: Vhost`.

Reply:
I've just tried: also in F30, doing the customization before starting the server makes rabbitmq-server.service unable to start. Instead starting the service first and then doing the customization and reload/restart the service works.